main

```diff
@@ -1,53 +1,27 @@
 #!/usr/bin/env ruby
 require_relative 'app'
+require_relative 'menu'
```

Comment:
- [ ] Kindly rename the file to main.rb, in hurry I think your missed to name file with .rb extension. Kindly do that in order to run the app smoothly.

Reply:
Well resolved. Good refactor and  App works fine.